services/movie.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
```

Comment:
```suggestion
genres_ids: Optional[list[int]] = None, 
``` use `Optional` here and all of the code if argument is optional